django_project/core/base_static/css/stylesheet.css

```diff
@@ -697,3 +706,8 @@ input[type='email']:hover::placeholder {
     background-size: contain;
     display: inline-flex;
 }
+
+.noplr {
```

Comment:
you could use bootstrap class: `pl-0 pr-0`
https://getbootstrap.com/docs/4.3/utilities/spacing/

Reply:
Those classes didn't work, unsure why.